lib/Sema/TypeCheckAttr.cpp

```diff
@@ -1145,6 +1145,8 @@ visitDynamicCallableAttr(DynamicCallableAttr *attr) {
 static bool hasSingleNonVariadicParam(SubscriptDecl *decl,
                                       Identifier expectedLabel,
                                       bool ignoreLabel = false) {
+  // FIXME(InterfaceTypeRequest): Remove this.
+  (void)decl->getInterfaceType();
```

Comment:
`index->hasInterfaceType()` few lines below needs this.

Reply:
After #27922 you can delete this.

Reply:
Indeed. Thanks!